test/cli.js

```diff
@@ -21,11 +21,15 @@ async function run(argv) {
       }
       return line;
     });
-  const globPattern = `**/*${argv.testFilePattern}*.test.{js,ts,tsx}`;
-  const spec = glob.sync(globPattern, {
-    cwd: workspaceRoot,
-    ignore,
-  });
+  const globPattern = `**/*${argv.testFilePattern}*`;
```

Comment:
```suggestion
  const globPattern = `**/*${argv.testFilePattern.replace(/\\/g, '/')}*`;
```

Reply:
This is a separate issue, no?

Reply:
It depends on how you look at it, I guess. You mentioned being able to run tests in VSCode debugger and it doesn't always work. But on the other hand, it's not that the change you made breaks anything.

I'm not very precious about whether the fix lands in this PR or a separate one. It'll just be nice to actually have it.

Reply:
>  But on the other hand, it's not that the change you made breaks anything.

That's the important part. Just because you are unable to run some task that's not checked in, doesn't mean everybody else shouldn't be able to.